airbyte-integrations/connectors/source-jira/integration_tests/full_configured_catalog.json

```diff
@@ -7455,10 +7455,10 @@
         },
         "supported_sync_modes": ["incremental"],
         "source_defined_cursor": true,
-        "default_cursor_field": ["created"]
+        "default_cursor_field": ["updated"]
```

Comment:
this is not all configured catalogs

Reply:
@po3na4skld did you check, if "updated" field existed for Issues?

Reply:
Yes

Reply:
Some additional [information](https://support.atlassian.com/jira-software-cloud/docs/advanced-search-reference-jql-fields/#Advancedsearchingfieldsreference-UpdatedUpdatedDateupdatedDateUpdated)